src/app/(app)/members/components/DetailMemberHeader.tsx

```diff
@@ -0,0 +1,53 @@
+"use client";
+
+import React, { useEffect, useState } from "react";
+import { Sidebar } from "@/app/layout/components/Sidebar";
+import { Button } from "@/components/ui/button";
+import { HamburgerMenuIcon } from "@radix-ui/react-icons";
+import { usePathname, useRouter } from "next/navigation";
+import { Routes } from "@/lib/routes";
+import { DetailMemberBreadCrumbs } from "./DetailMemberBreadCrumbs";
+
+export const DetailMemberHeader = ({ id }: { id: string }) => {
+  const pathname = usePathname();
+  const router = useRouter();
+
+  const [isSidebarOpen, setSidebarOpen] = useState<boolean>(false);
+
+  const toggleSidebar = () => {
+    setSidebarOpen(!isSidebarOpen);
+  };
+
+  useEffect(() => {
```

Comment:
no need of this `useEffect` you can handle through breadcrums

Reply:
fixe